lib/rules/block-scoped-var.js

```diff
@@ -316,3 +316,5 @@ module.exports = function(context) {
     };
 
 };
+
+module.exports.schema = [];
```

Comment:
Yep, that's exactly what it does if there's no schema. In this case, the schema specifies that `block-scoped-var` does not accept any options, so if any are supplied, it will warn that there are too many.